page-changelog.php

```diff
@@ -15,6 +15,108 @@
+         <li>User names must now be at least 3 characters long - <a data-cke-saved-href="https://github.com/owncloud/core/issues/29237" href="https://github.com/owncloud/core/issues/29237">#29237</a></li>
+         <li>AccountMapper get by email is now case insensitive - <a data-cke-saved-href="https://github.com/owncloud/core/issues/29341" href="https://github.com/owncloud/core/issues/29341">#29341</a></li>
+         <li>Remove deprecated federated share API warning as it needlessly pollutes logs - <a data-cke-saved-href="https://github.com/owncloud/core/issues/29364" href="https://github.com/owncloud/core/issues/29364">#29364</a></li>
+         <li>Improve UI for public link sharing permissions for folders - <a data-cke-saved-href="https://github.com/owncloud/core/issues/29413" href="https://github.com/owncloud/core/issues/29413">#29413</a></li>
+         <li>Replace notify user for local shares with button - <a data-cke-saved-href="https://github.com/owncloud/core/issues/29463" href="https://github.com/owncloud/core/issues/29463">#29463</a></li>
+         <li>Log out current user after submitting form in password reset page - <a data-cke-saved-href="https://github.com/owncloud/core/issues/29464" href="https://github.com/owncloud/core/issues/29464">#29464</a></li>
+         <li>Update minimum supported browser versions - <a data-cke-saved-href="https://github.com/owncloud/core/issues/29507" href="https://github.com/owncloud/core/issues/29507">#29507</a></li>
+         <li>Admins can now change display name even when its modification is disallowed for regular users - <a data-cke-saved-href="https://github.com/owncloud/core/issues/29442" href="https://github.com/owncloud/core/issues/29442">#29442</a><br></li>
+      </ul>
+      <h4><a data-cke-saved-href="https://github.com/owncloud/core/blob/stable10/CHANGELOG.md#removed" href="https://github.com/owncloud/core/blob/stable10/CHANGELOG.md#removed" id="user-content-removed" class="anchor"></a>Removed<br></h4>
+      <ul>
+         <li>Remove AvatarPermissions repair step - <a data-cke-saved-href="https://github.com/owncloud/core/issues/29202" href="https://github.com/owncloud/core/issues/29202">#29202</a></li>
+         <li>Remove unused FTP code - <a data-cke-saved-href="https://github.com/owncloud/core/issues/29186" href="https://github.com/owncloud/core/issues/29186">#29186</a></li>
+         <li>Remove app store related code obsoleted by market app - <a data-cke-saved-href="https://github.com/owncloud/core/issues/29249" href="https://github.com/owncloud/core/issues/29249">#29249</a></li>
+      </ul>
+      <h4><a data-cke-saved-href="https://github.com/owncloud/core/blob/stable10/CHANGELOG.md#fixed" href="https://github.com/owncloud/core/blob/stable10/CHANGELOG.md#fixed" id="user-content-fixed" class="anchor"></a>Fixed<br></h4>
```

Comment:
It seems the magic tool is also referencing the original file on Github. Not sure if we want to keep this.